cheatsheets/XML_External_Entity_Prevention_Cheat_Sheet.md

```diff
@@ -549,6 +549,24 @@ libxml_disable_entity_loader(true);
 
 A description of how to abuse this in PHP is presented in a good [SensePost article](https://www.sensepost.com/blog/2014/revisting-xxe-and-abusing-protocols/) describing a cool PHP based XXE vulnerability that was fixed in Facebook.
 
+# PYTHON
+
+The following information for XXE injection in PYTHON 3 is directly from this [official PYTHON 3 documentation page](https://docs.python.org/3/library/xml.html#xml-vulnerabilities).
+
+The below table gives an overview of various modules in PYTHON 3 used for XML parsing and whether they are vulnerable or not.
+
+| Attack Type               | sax        | etree      | minidom    | pulldom    | xmlrpc     |
+|---------------------------|------------|------------|------------|------------|------------|
+| Billion Laughs            | Vulnerable | Vulnerable | Vulnerable | Vulnerable | Vulnerable |
+| Quadratic Blowup          | Vulnerable | Vulnerable | Vulnerable | Vulnerable | Vulnerable |
+| External Entity Expansion | Safe       | Safe       | Safe       | Safe       | Safe       |
+| DTD Retrieval             | Safe       | Safe       | Safe       | Safe       | Safe       |
+| Decompression Bomb        | Safe       | Safe       | Safe       | Safe       | Vulnerable |
+
+For details on PYTHON 2, refer this [official PYTHON 2 documentation page](https://docs.python.org/2/library/xml.html#xml-vulnerabilities). Kindly note that PYTHON 2 End of Life has been announced for January 1st 2020. After that PYTHON 2 will become obsolete and no official support will be available.
+
+Currently there are no explicit methods available for above mentioned python modules to mitigate the vulnerability. Hence sanitizes input prior to processing using the above libraries. 
```

Comment:
`for the above-mentioned Python` ... to mitigate `these vulnerabilities`.

Hence `you must sanitize`.

Reply:
Note that the second sentence is ambiguous, it either means `"you should use the above libraries to sanitize your input" and then process it ...` or `"you should use something else to sanitize your input" and then take that sanitized data and feed it to one of these libraries for further processing`.

I know what you're trying to say. But, you've failed to unambiguously state it. And someone should fix this before it's merged.